server/channels/db/migrations/postgres/000105_remove_tokens.up.sql

```diff
@@ -4,8 +4,7 @@ WITH oauthDelete AS (
 	DELETE FROM oauthaccessdata o
 	WHERE NOT EXISTS (
 		SELECT p.* FROM preferences p
-		WHERE o.clientid = p.name AND o.userid = p.userid AND p.category = 'oauth_app' 
-		and p.name IS NULL
+		WHERE o.clientid = p.name AND o.userid = p.userid AND p.category = 'oauth_app'
```

Comment:
Has this migration already been applied to the cloud servers?

Reply:
Yes, Cloud and 7.10

Reply:
So in that case, we cannot change this. Because this won't run again.

Reply:
Well this is an adjustment for instances where they haven't upgraded and run this faulty query, ideally they upgrade to the dot version from this fix? I assume this will go into a dot version?

Reply:
@agnivade Can I mark this conversation as resolved?

Reply:
Apologies I missed this. What happens to those servers where there this has already been applied?

Reply:
For the servers where this has already been applied, `000108_remove_orphaned_oauth_preferences` will be used to clean up the orphaned preferences. If you didn't run the faulty migration then `000108_remove_orphaned_oauth_preferences` just won't have anything to delete

Reply:
Thanks for that clarification!